cms/djangoapps/contentstore/management/commands/swap_from_auto_track_cohort_pilot.py

```diff
@@ -0,0 +1,168 @@
+from contentstore.course_group_config import GroupConfiguration
+from course_modes.models import CourseMode
+from django.core.management.base import BaseCommand, CommandError
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+from opaque_keys.edx.locator import LibraryUsageLocator
+
+from openedx.core.djangoapps.course_groups.cohorts import CourseCohort
+from openedx.core.djangoapps.course_groups.models import (CourseUserGroup, CourseUserGroupPartitionGroup)
+from openedx.core.djangoapps.verified_track_content.models import VerifiedTrackCohortedCourse
+
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.modulestore.django import modulestore
+from xmodule.partitions.partitions import ENROLLMENT_TRACK_PARTITION_ID
+from xmodule.partitions.partitions_service import PartitionService
+
+
+class Command(BaseCommand):
+    """
+    Migrates a course's xblock's group_access from Verified Track Cohorts to Enrollment Tracks
+    """
+
+    def handle(self, *args, **options):
+        # TODO convert to config model object configurable via django admin
+        course_id = 'course-v1:edX+DemoX+Demo_Course'
+        audit_cohort_names = ['Audit Cohort', 'Third Cohort']
+
+        module_store = modulestore()
+
+        try:
+            course_key = CourseKey.from_string(course_id)
+        except InvalidKeyError:
+            try:
+                course_key = SlashSeparatedCourseKey.from_string(course_id)
+            except InvalidKeyError:
+                raise CommandError("Invalid course_key: '%s'." % course_id)
+
+        items = module_store.get_items(course_key)
+        if not items:
+            raise CommandError("Course with %s key not found." % course_id)
+
+        # Get the CourseUserGroup IDs for the audit course names
+        audit_course_user_group_ids = CourseUserGroup.objects.filter(
+            name__in=audit_cohort_names,
+            course_id=course_key,
+            group_type=CourseUserGroup.COHORT,
+        ).values_list('id', flat=True)
+
+        # Get all of the audit CourseCohorts from the above IDs that are RANDOM
```

Comment:
Really we will need to know if they are random AND what content groups they are linked to. Because it could be unclear what to do if there is more than one random cohort linked to a content group, we might just want to obtain this information manually.